fixtures/autofix-all-resources-fixed_v1.yml

```diff
@@ -189,7 +189,8 @@ spec:
     spec:
       # Trust me the following line is longer than 80 characters
       containers:
-      - image: gcr.io/docker-images-directory/apps/production/fake-image-app:<%= current_sha %>
+      - image: gcr.io/docker-images-directory/apps/production/fake-image-app:<%= current_sha
```

Comment:
Is there a way to not do the 80 (or whatever it is) character limit?

Reply:
I doesn't look like it's supported right now but I could open a feature request